.changeset/thin-snails-compete.md

```diff
@@ -0,0 +1,15 @@
+---
+'@powersync/service-core': minor
+'@powersync/service-module-mssql': minor
+'@powersync/service-module-postgres-storage': patch
+'@powersync/service-module-mongodb-storage': patch
+'@powersync/service-module-postgres': patch
+'@powersync/service-errors': patch
+'@powersync/service-module-mysql': patch
+'@powersync/service-image': patch
```

Comment:
Should `@powersync/service-image` also be a minor bump? I believe that should have the same version as `@powersync/service-core`